src/modules/rppt_tensor_effects_augmentations.cpp

```diff
@@ -921,7 +1006,23 @@ RppStatus rppt_gaussian_noise_gpu(RppPtr_t srcPtr,
 
 /******************** non_linear_blend ********************/
 
-RppStatus rppt_non_linear_blend_gpu(RppPtr_t srcPtr1,
+RppStatus rppt_non_linear_blend_host(RppPtr_t srcPtr1,
```

Comment:
I think the merge did not happen properly
Please remove the api call for rppt_non_linear_blend_host since it's already added at L417

Reply:
Done